src/scss/nhsuk/_index.scss

```diff
@@ -0,0 +1,5 @@
+// Settings
+@import "src/scss/nhsuk/colours";
```

Comment:
do we need the full-path? i.e would import 'colours' work? ignore if it all imports as is anyways

Reply:
good spot cheers! have resolved